multi_import/formats.py

```diff
@@ -93,6 +93,8 @@ def __init__(self):
 
     @classmethod
     def ensure_unicode(cls, file_contents):
+        if not isinstance(file_contents, bytes):
```

Comment:
I was not sure how to tackle this method `ensure_unicode` so I added a quick if statement so that the method returns file_contents as it is if file_contents is unicode. (In Python 3, unicode str is the default, and str and bytes have been separated. Further reference about unicode, str, bytes in Python 3: https://medium.com/better-programming/strings-unicode-and-bytes-in-python-3-everything-you-always-wanted-to-know-27dc02ff2686)

Reply:
Use `six.text_type` instead of `bytes` - to maintain Python 2 back-compatibility.

Reply:
@sibartlett Thank you, Simon for the feedback! I think using `six` is a good idea! I realized that `six` also has a method to ensure unicode() (in python 2) and str (in python 3). I'll ping you in the code for your feedback.

Reply:
Updated by using `six.text_type` as Simon suggested.